tests/ckeditor/basepath.js

```diff
@@ -31,6 +32,20 @@ describe( 'basePath', function() {
 		window.CKEDITOR_BASEPATH = 'http://foo.com/ckeditor/';
 		expect( CKEDITOR._getBasePath() ).to.equal( 'http://foo.com/ckeditor/' );
 	} );
+
+	function testGetBasePathFromTag( url, expectedBasePath ) {
```

Comment:
Even though it's cool, DRY, KISS and everything else, it will break Bender.js, because you can't filter out test cases created on the fly. So no support for regressions and single test runs...